web/client/components/map/leaflet/MeasurementSupport.jsx

```diff
@@ -55,6 +55,9 @@ L.getMeasureWithTreshold = (value, threshold, source, dest, precision, sourceLab
 */
 
 L.GeometryUtil.readableDistance = (distance, isMetric, isFeet, isNauticalMile, precision, options) => {
+    if (options === null) {
```

Comment:
This way, you are still breaking things.
You should mantain non-overridden behaviour by default. I think the best way is to provide a default for options that triggers the old behaviour.